docs/maven-example.md

```diff
@@ -0,0 +1,74 @@
+# Fabric8 Maven Plugin + Kompose:
+Lets deploy a Springboot Java application with Docker Compose file using Fabric8 Maven Plugin to Kubernetes or OpenShift.
+##### Requirements
+* Linux or MacOS or Windows
+* JDK 1.7+ - [JDK Quick Installation Guide](http://openjdk.java.net/install/)
+* Maven 3.x+ - [Maven Installation Guide](https://maven.apache.org/install.html) 
+* Kompose
```

Comment:
no link to kompose

Reply:
Is it really needed? as the tutorial is on Kompose.

Reply:
Yes. Other ones have links. May as well add one. It's good for SEO anyways.